pkg/array/array.go

```diff
@@ -137,6 +137,8 @@ type PowerStoreArray struct {
 	Insecure      bool                 `yaml:"skipCertificateValidation"`
 	IsDefault     bool                 `yaml:"isDefault"`
 	NfsAcls       string               `yaml:"nfsAcls"`
+	MetroTopology string               `yaml:"metroTopology"`
+	Labels        map[string]string    `yaml:"labels"`
```

Comment:
Do we need to update the sample [here](https://github.com/dell/csi-powerstore/blob/main/samples/secret/secret.yaml)?

Reply:
Its added now